cmd/serve.go

```diff
@@ -227,8 +228,22 @@ func buildAPIDependencies(
 		)
 		logger.Info("mailer enabled", "host", cfg.App.Mailer.SMTPHost, "port", cfg.App.Mailer.SMTPPort)
 	}
+
+	wconfig := &webauthn.Config{
+		RPDisplayName: cfg.App.Authentication.PassKey.RPDisplayName,
+		RPID:          cfg.App.Authentication.PassKey.RPID,
+		RPOrigins:     cfg.App.Authentication.PassKey.RPOrigins,
+	}
+	webAuthConfig, err := webauthn.New(wconfig)
+	if err != nil {
+		if wconfig.RPDisplayName == "" && wconfig.RPID == "" && wconfig.RPOrigins == nil {
+			webAuthConfig = &webauthn.WebAuthn{}
```

Comment:
we can store `nil` here and in supported strategies method of `service.go` we can check if config is `nil`?

Reply:
done